progenitor/tests/build_propolis.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 Oxide Computer Company
+
+// ensure that the websocket channel used for serial console compiles.
```

Comment:
we might want to see the output as well along these lines:

https://github.com/oxidecomputer/progenitor/blob/main/progenitor-impl/tests/test_output.rs#L48

I know it's a little silly that we have two tests for each file, but one is for examining output and the other is just to make sure it compiles (with some spot checks).